dashboard/app/models/pd/application/teacher_application.rb

```diff
@@ -399,7 +398,7 @@ def principal_approval_state
       response = Pd::Application::PrincipalApprovalApplication.find_by(application_guid: application_guid)
       return PRINCIPAL_APPROVAL_STATE[:complete] + response.full_answers[:do_you_approve] if response
 
-      principal_approval_email = emails.where(email_type: 'principal_approval').order(:created_at).last
+      principal_approval_email = emails.where(email_type: 'admin_approval').order(:created_at).last
```

Comment:
Would this affect any existing rows in the database? ie would emails accidentally get queued again?

Reply:
Good question –– this doesn't queue emails (just looks at existing emails), but it will affect what shows up in detail view.

We show this
<img width="1089" alt="image" src="https://user-images.githubusercontent.com/9142121/203354296-480376b1-6aa6-4c0a-aaee-d8db6d422526.png">
when a principal approval email has been sent but not submitted, e.g. https://adhoc-acq-teacher-applications-studio.cdn-code.org/pd/application_dashboard/csd_teachers/48432.

This will mess up *existing* principal approvals that are incomplete, e.g. https://studio.code.org/pd/application_dashboard/csd_teachers/48114. They'd look like the emails were never sent.

I could go two ways on this:
1) We do not have any principal approval emails currently for this year, so updating this now wouldn't break apps that RPs would actually look at.
2) Since RPs still can access old apps in detail view, keeping these consistent might be nice. So, I could query for both `principal_approval` and `admin_approval` so that old apps don't break.
I'm leaning towards (2) since it wouldn't be a hard fix and so that old apps don't break. Do you have thoughts?

Reply:
Regional partners can't access last years apps so I don't think we need to worry about #2

Reply:
If old apps aren't accessible and no emails have been sent or queued for this year, I think this looks good to update now